Added coefficients for CIGS and a-Si modules types.

MLEEFS opened the pull request:
Hi PVLib,

I added coefficients for CIGS and a-Si modules types. I also provided references to document with Spectral Reponses (SRs) of the CIGS and a-Si modules. Reference also includes additional testing and field validation of the model.

wholmgren commented on pvlib/atmosphere.py:
```diff
@@ -374,10 +374,13 @@ def first_solar_spectral_correction(pw, airmass_absolute, module_type=None,
             * 'monosi', 'xsi' - First Solar TetraSun modules.
             * 'multisi', 'polysi' - multi-crystalline silicon modules.
+            * 'cigs' - anonymous copper indium gallium selenide PV module
+	    * 'asi' - anonymous amorphous silicon PV module
```
The 'asi' line has a tab instead of 4 spaces.

MLEEFS replied:
Hi Will. Sorry for the long delay. I tried making the recommended changes. Please confirm whether they were made successful and if you need me to make any additional changes.

wholmgren commented on pvlib/atmosphere.py:
```diff
@@ -450,6 +457,11 @@ def first_solar_spectral_correction(pw, airmass_absolute, module_type=None,
+        0.85252, -0.022314, -0.0047216, 0.13666, 0.013342, -0.0008945)
+    _coefficients['asi'] = (
+        1.12094, -0.047620, -0.0083627, -0.10443, 0.098382,-0.0033818)
+	
```
This empty-looking line has a tab character. You can just delete the whole line.

wholmgren commented:
Thanks Mitch, the tests look great. Still need to remove the tab from the 'asi' line in the documentation, though. It currently reads...

```
(tab)(4 spaces)* 'asi' - anonymous amorphous silicon PV module
```

but needs to read...

```
[12 spaces]* 'asi' - anonymous amorphous silicon PV module
```

If you can make this change in the next day or two then we can include it in the 0.4.4 release.

MLEEFS commented:
Hi Will. I changed the tab to spaces. Sorry about that. I modified the settings in my text editor, so this should no longer be an issue.  

Quick question. For the cdte, monosi, and polysi coefficients fed into test_first_solar_spectral_correction, why don't they perfectly match what I get when running the provided ams and pws inputs?

wholmgren commented:
No worries, I think everyone gets tripped up by tabs vs. spaces in Python at some point. 

I think the discrepancy is due to the fact that I implemented the algorithm with a preliminary set of coefficients in commit 28cc9fb, then you updated the coefficients in 58d77ea2 but did not update the expected output. The changes to the coefficients must have been small enough that they did not change the output by more than the test's tolerance allowed. Do you want to update them or should we go ahead and merge this? Either way is fine with me.

MLEEFS commented:
Thanks for being patient with my tab issues.

I will update the coefficients. I should commit those by the end of the day.

MLEEFS commented:
Updated M values for unit testing of first_solar_spectral_correction. Hopefully things are good to go. Let me know if any more changes need to be made.

wholmgren commented:
Thanks Mitch!
